src/Core/src/Handlers/View/ViewHandlerOfT.Android.cs

```diff
@@ -11,6 +13,12 @@ public partial class ViewHandler<TVirtualView, TNativeView> : INativeViewHandler
 		View? INativeViewHandler.NativeView => (View?)base.NativeView;
 		public Context? Context => MauiContext?.Context;
 
+		protected Context ContextWithValidation([CallerMemberName] string callerName = "")
```

Comment:
This way we don't have to keep checking for null everywhere we know that it shouldn't be